tests/test_core_utils.py

```diff
@@ -24,3 +26,17 @@ def test_accessing_hydra_config(hydra_restore_singletons: Any) -> Any:
         del cfg["hydra"]
     assert cfg.job_name == "UNKNOWN_NAME"
     assert cfg.config_name == "accessing_hydra_config"
+
+
+def test_py_version_resolver(hydra_restore_singletons: Any, monkeypatch: Any) -> Any:
+    Version = namedtuple("Version", "major minor micro")
+    with monkeypatch.context() as m:
+        m.setattr(sys, "version_info", Version(3, 7, 2))
+        OmegaConf.clear_resolvers()
```

Comment:
what happens to the resolvers after this function finishes?

Reply:
for now I suggest that you copy [this](https://github.com/omry/omegaconf/blob/master/tests/conftest.py#L9) to automatically restore the resolvers.

It would probably make sense to introduce it as a pytest plugin in OmegaConf similar to the pytest plugin in Hydra.

Reply:
> what happens to the resolvers after this function finishes?

Since OmegaConf won't let you overwrite the resolver, the test fails if the the resolver has already been set by other tests (the monkeypatch won't take effect), which is why i try to clear and reset the resolvers after the tests are done.

> It would probably make sense to introduce it as a pytest plugin in OmegaConf similar to the pytest plugin in Hydra.

I'm not sure what you mean here: import the fixture from omegaconf into hydra pytest_plugin.py somehow and call it?

Or do you mean copy the fixture from to ```hydra/extra/pytest_plugin.py``` and use it?

Reply:
Tempoary: copy the code.
Real solution, send a PR against OmegaConf introducing a pytest plugin like the one I added to Hydra.
This would allow any pytest user that has OmegaConf install to use this fixture.

Reply:
sounds good, thx for clarifying